modules/cp4ba/templates/operator_shared_pv.yaml.tmpl

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: operator-shared-pv
+  namespace: ${cp4ba_project_name}
+spec:
+  accessModes:
+  - ReadWriteMany
+  capacity:
+    storage: 1Gi
+  nfs:
+    path: /shared/operator
+    server:
+  persistentVolumeReclaimPolicy: Retain
```

Comment:
Change this to `delete`.  We don't want to retain storage space when the clusters are provisioned.

Reply:
Changed it to: `persistentVolumeReclaimPolicy: Delete`